configuration/config-txt/camera.md

```diff
@@ -4,8 +4,7 @@
 
 Setting `disable_camera_led` to `1` prevents the red camera LED from turning on when recording video or taking a still picture. This is useful for preventing reflections when the camera is facing a window, for example.
 
-
-
+Setting `awb_auto_is_greyworlds`to `1` allows libraries or applications that doesn't support greyworld to capture valid frames and videos with NoIR cameras. It switches "auto" awb mode to select "greyworld" instead. This should be needed for NoIR cameras only, but is needed in case of [Removing the infrared (IR) filter from the Raspberry Pi High Quality Camera](../../hardware/camera/hqcam_filter_removal.md) as well.
```

Comment:
"that doesn't support" -> ""that don't support"

"frames" -> "images" ? :man_shrugging: 

Second sentence feels a bit clunky? What about "This should only be needed for NoIR cameras, or when you've [removed the infrared (IR) filter](../../hardware/camera/hqcam_filter_removal.md) from the Raspberry Pi High Quality Camera." ?

Reply:
all 3 fine for me -- I am not a native speaker